polybar/debian/changelog

```diff
@@ -1,3 +1,9 @@
+polybar (3.4.3-0ubuntu0) focal; urgency=medium
```

Comment:
The changelog entry will be auto generated by the `gbp` command.  Sorry I didn't mention that.  Can you revert your edits to this file?

Reply:
Done n done!